.github/workflows/lint_go-vet.yml

```diff
@@ -0,0 +1,24 @@
+name: lint_go-vet
+on:
+  push:
+    branches:
+      - master
+      - main
+  pull_request:
+
+permissions:
```

Comment:
This perms line is not needed

Reply:
Done based on the current `linter.yml` we use for fiber: https://github.com/gofiber/fiber/blob/fe65ff8dff718db4dfb6921108371c9e248ee208/.github/workflows/linter.yml#L10-L11